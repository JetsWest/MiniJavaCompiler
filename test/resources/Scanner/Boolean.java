boolean b = false;
if (!b) {
    boolean c = true;
}