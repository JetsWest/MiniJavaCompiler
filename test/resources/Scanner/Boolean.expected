BOOL ID(b) BECOMES FALSE SEMICOLON IF LPAREN NOT ID(b) RPAREN LBRACE BOOL ID(c) BECOMES TRUE SEMICOLON RBRACE